karmada-duplication-deploy-example/index.json

```diff
@@ -0,0 +1,36 @@
+{
+    "title": "deploy workload through karmada duplicated type",
```

Comment:
The title is based on technology, perhaps it would be better to use a description that reflects value.

Such as
```
HA workload deployment
```